frontend/stylesheets/pages/_co-op.scss

```diff
@@ -0,0 +1,16 @@
+.co-op-description {
+  margin-bottom: 25px;
+}
+
+.co-op-group {
+  margin-top: 25px;
+}
+
+.co-op-option {
+  padding-left: 10px;
+}
+
+.radio-inline input[type='radio'], .radio input[type='radio'], input[type='radio'] {
```

Comment:
This is not how this should be selected. It also makes the linter unhappy. 

I will figure this out later. _(or if someone wants to fix it, I am also fine with that)_